README.md

```diff
@@ -1,6 +1,16 @@
 # Real User Data SDK
+## Overview
+This project is meant to help capture states and actions (in progress) of the user from web applications and websites. On top of the data relating to the user behavious and his user journey, the SDK can capture various application and technical data for in depth monitoring of the real world application. 
```

Comment:
"On top of the data relating to the user behavious and his user journey, the SDK can capture various application and technical data for in depth monitoring of the real world application."

This is a key part of the concept and I think we need to be additionally clear in wording.  Perhaps we could try something like the following?

"The Real User Data SDK facilitates the collection of non-PII (Personally Identifiable Information) such as customer behaviour, device specifications, performance timing, software release versions, and business impact (revenue, conversion).  This data is processed by our cloud-based platform to generate a modern class of aggregated data insights.  In particular, these insights drive decision making and test automation within Eggplant's suite of AI-assisted end-to-end user testing known as Eggplant DAI. We refer to this process as CXO (Customer Experience Optimisation)."